tools/testsys-config/src/lib.rs

```diff
@@ -261,6 +261,14 @@ pub struct GenericVariantConfig {
     pub control_plane_endpoint: Option<String>,
     /// The path to userdata that should be used for Bottlerocket launch
     pub userdata: Option<String>,
+    /// The directory containing Bottlerocket images. For metal, this is the directory containing
+    /// gzipped images.
+    pub os_image_dir: Option<String>,
+    /// The hardware that should be used for provisioning Bottlerocket. For metal, this is the
+    /// hardware csv that is passed to EKS Anywhere.
+    pub hardware_csv: Option<String>,
+    /// The tinkerbell ip used for provisioning Bare Metal EKS Anywhere clusters.
+    pub tinkerbell_ip: Option<String>,
```

Comment:
Can this also be something like `mytinkerbell.my.local.domain`? If so the name could be adjusted.

Reply:
The tinkerbell ip is not needed for workload cluster creation, so I've removed it.